### PR TITLE
4296 Tasks

### PR DESCRIPTION
**Pull request by wbyoko**
* Navigate to case detail after serving
* reviewedAt should show up
* add success message
* handle refresh on review page
* mailing date is an open text box

### CODE REVIEW

File: web-client/src/presenter/actions/StartCaseInternal/getServeToIrsAlertSuccessAction.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * get alert message when a petition is served
+ *
+ * @returns {object} the prop of the alert success message
+ */
+export const getServeToIrsAlertSuccessAction = () => {
+  return {
+    alertSuccess: {
+      message: 'The case has been created and served to the irs.',
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/StartCaseInternal/getServeToIrsAlertSuccessAction.js]: maybe `to the IRS` ?

File: web-client/src/presenter/sequences/serveToIrsSequence.js [hunk 2]
```diff
@@ -22,10 +25,13 @@ export const serveToIrsSequence = [
...
       setCaseAction,
       setPetitionIdAction,
+      setDocumentIdAction,
       closeFileUploadStatusModalAction,
       sendPetitionToIRSHoldingQueueAction,
       runBatchProcessAction,
-      gotoDocumentDetailSequence,
+      getServeToIrsAlertSuccessAction,
+      setAlertSuccessAction,
+      navigateToCaseDetailAction,
```
Comment (kkoskelin) [inline web-client/src/presenter/sequences/serveToIrsSequence.js]: You might try using `utilities/sequenceHelpers{showProgressSequenceDecorator}`to ensure we indicate progress occurring here.

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/sequences/serveToIrsSequence.js (hunk 2 above)
Comment (wbyoko) [reply web-client/src/presenter/sequences/serveToIrsSequence.js]: yup

File: web-client/src/presenter/actions/StartCaseInternal/getServeToIrsAlertSuccessAction.js (hunk 1 above)
Comment (wbyoko) [reply web-client/src/presenter/actions/StartCaseInternal/getServeToIrsAlertSuccessAction.js]: good catch!

**Pull request merged**